src/style.css

```
@@ -1,3 +1,74 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
```

Don't need to use fixed width for header
```suggestion
  width: 1005;
  padding: 0 50px;
  box-sizing: border-box;
```